src/layouts/SiteLaunchPad/components/SiteLaunchInfoGathering.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { Box, Button, Icon, Text, RadioGroup } from "@chakra-ui/react"
+import { Input, Radio } from "@opengovsg/design-system-react"
+import { useState } from "react"
+import { useForm } from "react-hook-form"
+import { BiRightArrowAlt } from "react-icons/bi"
+
+import { SITE_LAUNCH_PAGES } from "types/siteLaunch"
+
+import { SiteLaunchPadBody } from "./SiteLaunchPadBody"
+import { SiteLaunchPadTitle } from "./SiteLaunchPadTitle"
+
+export const SiteLaunchInfoCollectorTitle = (): JSX.Element => {
+  const title = "Tell us more about your domain"
+  const subTitle = "This will inform the steps you have to take for launch"
+  return <SiteLaunchPadTitle title={title} subTitle={subTitle} />
+}
+
+interface SiteLaunchInfoGatheringBodyProps {
+  setPageNumber: (number: number) => void
+}
+
+interface SiteLaunchFormData {
+  domain: string
+  nature: "new" | "live"
```

Comment:
nit: prefer have a const exporting this as namespaced values (const SiteNature: { New: "new", Live: "live" } are a nicer dev exp than typing strings

Reply:
82e17d1e1e7b0f464192c467d0ef1315ed8c8c3f